everyvoice/run_tests.py

```diff
@@ -5,9 +5,9 @@
 
 import argparse
 import importlib
-import os
 import re
 import sys
+from os.path import dirname
```

Comment:
this is fine, but not a necessary re-factor I don't think.

Reply:
I didn't like writing `os.path.dirname(os.path.dirname(__file__))` so that was my choice.

Reply:
Given how many instances there were, I got annoyed and changed the import.